test2

```diff
@@ -54,10 +54,19 @@ function drop_cache() {
 	echo 3 > /proc/sys/vm/drop_caches
 }
 
+function wait_umount() {
+	set +e
```

Comment:
The `set +e/-e` is not required, the `e` flag does not apply for commands that are executed as part of the `while` condition